Common/DataModel/CollisionAssociation.h

```diff
@@ -33,6 +33,29 @@ DECLARE_SOA_TABLE(TrackAssoc, "AOD", "TRACKASSOC", //! Table for track-to-collis
 
 DECLARE_SOA_TABLE(TrackCompColls, "AOD", "TRACKCOMPCOLLS", //! Table with vectors of collision indices stored per track
                   track_association::CollisionIds);
+
+namespace fwdtrack_association
+{
+DECLARE_SOA_INDEX_COLUMN(Collision, collision);            //! Collision index
+DECLARE_SOA_INDEX_COLUMN(FwdTrack, fwdtrack);              //! FwdTrack index
+DECLARE_SOA_INDEX_COLUMN(MFTTrack, mfttrack);              //! MFTTrack index
+DECLARE_SOA_ARRAY_INDEX_COLUMN(Collision, compatibleColl); //! Array of collision indices
+} // namespace fwdtrack_association
+
+DECLARE_SOA_TABLE(FwdTrackAssoc, "AOD", "FWDTRACKASSOC", //! Table for fwdtrack-to-collision association
+                  fwdtrack_association::CollisionId,
+                  fwdtrack_association::FwdTrackId);
+
+DECLARE_SOA_TABLE(FwdTrackCompColls, "AOD", "FWDTRACKCOLLS", //! Table with vectors of collision indices stored per fwdtrack
+                  fwdtrack_association::CollisionIds);
+
+DECLARE_SOA_TABLE(MFTTrackAssoc, "AOD", "MFTTRACKASSOC", //! Table for mfttrack-to-collision association
+                  fwdtrack_association::CollisionId,
+                  fwdtrack_association::MFTTrackId);
+
+DECLARE_SOA_TABLE(MFTTrackCompColls, "AOD", "MFTTRKCOMPCOLLS", //! Table with vectors of collision indices stored per mfttrack
```

Comment:
```suggestion
DECLARE_SOA_TABLE(MFTTrackCompColls, "AOD", "MFTTRKCOMPCOLL", //! Table with vectors of collision indices stored per mfttrack
```

Reply:
I guess you shortened it because it has a 14 (?) character limit. However, if first and third argument are not the same (except the s), then we can never write the table to a file which is probably a bad idea...